files-community/Files
# Feature: Updated tag icon in the Tags widget

**Resolved / Related Issues**
   Closes #issue...

**Validation**
How did you test these changes?
   1. Enable Tags widget
   2. Confirm it's using the new icon

**Screenshots (optional)**
![image](https://github.com/files-community/Files/assets/39923744/a9f21d75-f63b-40e4-a424-316217633ceb)
